<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadGamePage</class>
 <widget class="QWidget" name="LoadGamePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="MainButton" name="sureBtn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>320</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="MainButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>320</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioBtnAI">
   <property name="geometry">
    <rect>
     <x>384</x>
     <y>263</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>与AI对抗</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioBtnOne">
   <property name="geometry">
    <rect>
     <x>138</x>
     <y>263</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>单人模式</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioBtnTwo">
   <property name="geometry">
    <rect>
     <x>261</x>
     <y>263</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>双人模式</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MainButton</class>
   <extends>QPushButton</extends>
   <header location="global">mainbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
